nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractPutElasticsearch.java

```diff
@@ -67,13 +69,28 @@ public abstract class AbstractPutElasticsearch extends AbstractProcessor impleme
         .required(true)
         .build();
 
+    static final PropertyDescriptor OUTPUT_ERROR_RESPONSES = new PropertyDescriptor.Builder()
+            .name("put-es-output-error-responses")
+            .displayName("Output Error Responses")
+            .description("If this is enabled, errors will be output to the \"error_responses\" relationship.")
```

Comment:
Are the other relationship(s) honored when this is enabled? I presume the original behavior is maintained and this is an extra feature that can be enabled for better error-handling. If so can you add documentation here that the other relationships' behaviors remain intact?

Reply:
Also might be worth mentioning that the relationship is auto-terminated by default so they also need to connect that relationship to a downstream processor.

Reply:
I've updated the property description to indicate that the (optional) output of responses messages from Elasticsearch is indepenedant of the output of Records (or FlowFiles) from these processors

I've changed the relationship from being autoTerminated and instead it will dynamically be added (and therefore need to be connected downstream) if this property is set to `true`